Adds an additionnal intialization possibility

fx-leb opened the pull request:
For now, extend the nuxeo official image is quite hard. The postgres official image proposes a simple mecanism. Here is the implementatation for nuxeo image 
- Adds an additionnal intialization possibility : scripts under /docker-entrypoint-initnuxeo.d are run before starting the service.

I also added two environments variables to make easiest to use this image to build a nuxeo cluster
- NUXEO_TRANSIENT_STORE
- NUXEO_BINARY_STORE

Commit:
NXIO-497 customize package origin

Commit:
- Adds an additionnal intialization possibility : scripts under /docker-entrypoint-initnuxeo.d are run before starting the service. This behavior is copied
- Adds two environnement variables to allow the use of this image in a cluster
  - NUXEO_TRANSIENT_STORE
  - NUXEO_BINARY_STORE

fx-leb commented:
The build failure does not appear to to be related to the PR.

dmetzler commented:
This is interesting. Before merging, modifications should be done on docker-entrypoint.sh and Dockerfile at the top level and then applied to 7.10 and 8.1 by launching `generateDockerfile.sh 7.10` and  `generateDockerfile.sh 8.1`.

Commit:
Sync this fork repository to keep it up to date with the  upstream repository

Merge remote-tracking branch 'upstream/master'

Commit:
Adds an additionnal intialization possibility : scripts under /docker-entrypoint-initnuxeo.d are run before starting the service. This behavior is copied
Adds two environnement variables to allow the use of this image in a cluster
      - NUXEO_TRANSIENT_STORE
      - NUXEO_BINARY_STORE

fx-leb commented:
I applied the changes to the templates. I then use the generator to update the dockerfile and entrypoint of all versions.

dmetzler commented:
Transient API exists as non experimental only since 7.10. So we should not update 6.0 and 7.4 entrypoint for the new transient environment variable. 
Also, please avoid doing formatting changes in a non dedicated commit.

Commit:
remove NUXEO_TRANSIENT_STORE environment variable from 6.0 and 7.4 since transient api was experimental in these versions

fx-leb commented:
I removed NUXEO_TRANSIENT_STORE environment variable from 6.0 and 7.4

fx-leb commented:
Hello,

When could we expect an update on docker hub ?

By the way, thank for the review.

Regards.

dmetzler commented:
You can follow integration here : https://github.com/docker-library/official-images/pull/1523
